pygmt/helpers/utils.py

```diff
@@ -306,5 +306,23 @@ def args_in_kwargs(args, kwargs):
     --------
     bool
         If one of the required arguments is in ``kwargs``.
+
+    Examples
+    --------
+
+    >>> args_in_kwargs(args=["A", "B"], kwargs={"C": "xyz"})
+    False
+    >>> args_in_kwargs(args=["A", "B"], kwargs={"B": "af"})
+    True
+    >>> args_in_kwargs(args=["A", "B"], kwargs={"B": None})
+    False
+    >>> args_in_kwargs(args=["A", "B"], kwargs={"B": True})
+    True
+    >>> args_in_kwargs(args=["A", "B"], kwargs={"B": False})
+    False
+    >>> args_in_kwargs(args=["A", "B"], kwargs={"B": 0})
+    True
     """
-    return any(arg in kwargs for arg in args)
+    return any(
+        kwargs.get(arg) is not None and kwargs.get(arg) is not False for arg in args
```

Comment:
Is the following code better?
```suggestion
        kwargs.get(arg) not in (None, False) for arg in args
```

Reply:
This doesn't work, need to use `is` instead of `in` or `==` to do the check against None or False:

```
323     >>> args_in_kwargs(args=["A", "B"], kwargs={"B": 0})
Expected:
    True
Got:
    False

```